microscope/monitor/monitor.py

```diff
@@ -8,6 +8,7 @@
 import queue as queuemodule
 import re
 
+import kubernetes
```

Comment:
Nitpick: you could `import kubernetes.client` here, since you are only using `client` in your code.